db/migrate/20190930145710_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
```

Comment:
It's a good idea to have your database constraints the same as the model constraints.  Because there are `presence: true` validations, I would change the migration to enforce that on the database level too